src/test/config.sol

```diff
@@ -98,10 +98,10 @@ contract Config {
         // Values for spell-specific parameters
         //
         spellValues = SpellValues({
-            deployed_spell:                 address(0xdA607F191B2D9a76Bc7648a1a62B43bb721668e2),        // populate with deployed spell if deployed
+            deployed_spell:                 address(0),        // populate with deployed spell if deployed
             deployed_spell_created:         1663172422,                 // use get-created-timestamp.sh if deployed
             previous_spell:                 address(0),        // supply if there is a need to test prior to its cast() function being called on-chain.
```

Comment:
if previous spell have not been cast yet, it can be set here to allow testing new spell actions (e.g. chainlog bump)

Reply:
I am getting ""This spell has already been scheduled" revert